### ISSUE TITLE
Broken help links (i.e. 404s)

### ISSUE THREAD

**Issue by dmontagu** [returntocorp/bento#152]
**Describe the bug**
It looks like the info links for many of the new checks are broken.

In particular, at least all of the checks starting with `B` or `A` have broken links.

(Also, it looks like B008 generates a malformed link, even if it did exist -- it includes a `:` on the end. Seems like maybe an upstream formatting inconsistency.)

**Expected result**
Describe what you expected to see.

Any displayed `lintlyci.github.io` link should not take me to a 404.

**Actual result**
When possible, provide verbose output (--verbose)

**Additional context**
bento-cli 0.3.0

**Comment by dmontagu** [comment]
Related issue: if you want to ignore the flake8 `B008`, you have to add it to the .bento.yml as
```
- "B008:"
```
due to the colon on the end (you get an error if you don't use quotes).

Might be worth submitting a fix upstream.

(The web framework I use ([fastapi](https://github.com/tiangolo/fastapi)) heavily violates B008 for dependency injection.)

**Comment by dmontagu** [comment]
This case is particularly frustrating:
```
306:13 error   Loop control variable 'label' not used within the loop body. If this is intended, start the name ... r2c.flake8     B007 https://lintlyci.github.io/Flake8Rules/rules/B007.html
```

I would be happy to fix it by following the instructions, but it is cut off before the end. So I try to go to the link for more details, and it is broken. (I'm assuming it's saying start with a `_`, but it would still be nice if there was a way to see the full description.)

**Comment by dlukeomalley** [comment]
Thanks for finding and reporting this @dmontagu. I'm queuing this up for discussion here, and see that we previously closed #83 (similar issue).

Created a new ticket for the `B008:` bug: #154

**Comment by pabloest** [comment]
I am running into this 404 link issue running Bento `0.7.0b7` with the check `r2c.flake8.B950`.
I ran Bento against this project: https://github.com/flaskbb/flaskbb

<img width="1154" alt="Screen Shot 2019-12-10 at 4 45 34 PM" src="https://user-images.githubusercontent.com/302941/70581608-90eb9e80-1b6c-11ea-94dc-ad957582bc0f.png">


Clicking on the hyperlink in the terminal, my browser opens: https://lintlyci.github.io/Flake8Rules/rules/B950.html
and this leads to a 404:

<img width="1420" alt="Screen Shot 2019-12-10 at 4 42 36 PM" src="https://user-images.githubusercontent.com/302941/70581550-5c77e280-1b6c-11ea-84b7-75079383832a.png">

**Comment by ulziibay** [comment]
Well we are gonna have to deploy our own rules repo for all non-r2c checks it seems :(

**Comment by ulziibay** [comment]
@minusworld lets queue this up for our tech debt sprint

**Comment by daghan** [comment]
Same for Flake8 B007
![image](https://user-images.githubusercontent.com/433961/70636909-2fb6e000-1beb-11ea-9607-80ddd4686a9c.png)

**Comment by minusworld** [comment]
Also related: https://github.com/returntocorp/bento-core/issues/379

**Comment by dlukeomalley** [comment]
Fixed in 0.8 by @nbrahms
